client/src/Components/Home/Home.js

```diff
@@ -7,7 +7,11 @@ const Home = () => {
   return (
     <section className='home'>
       <h1 className='logo-home'>BEAM</h1>
-      <h2>Thanks for stopping by</h2>
+      <h2>Beam is an interactive self-care feeling tracking app.</h2>
+        <h3 className='blurb'>
+        We created a flow intended to help you work through any kind of feeling or emotion you might be experiencing. You'll be asked to choose a feeling and reflect on it. Add anything about your experience that might help you avoid or recreate the feeling in the future (depending on the feeling). After your reflection, you'll see an offering of a few activities that might help. <br></br>Your entries will be available to you in the Feelings Archive for future reflection. 
```

Comment:
Love this blurb - great writing ❤️ 

I'm not sure that I love the current styling, but I also don't have any other way to style it at the moment lol. All that is to say, I'm open to changing it, too!

Reply:
I love the blurb, ++ to what Joe said. It _does_ feel like a huge chunk of info that my brain doesn't automatically want to read. I wonder if we broke it up into bullets if it would feel better?

Reply:
thanks y'all